rules/otp.bzl

```
@@ -102,6 +102,49 @@ otp_json = rule(
     },
 )
 
+def _otp_alert_digest_impl(ctx):
+    file = ctx.actions.declare_file("{}.json".format(ctx.attr.name))
+
+    outputs = [file]
+
+    inputs = [
+        ctx.file._opentitantool,
+        ctx.file.otp_img,
+    ]
+
+    ctx.actions.run(
+        outputs = outputs,
+        inputs = inputs,
+        arguments = [
+            "otp",
+            "alert-digest",
+            "--output",
+            file.path,
+            ctx.file.otp_img.path,
+            ctx.attr.lc_state,
+        ],
```

Could you switch the generation of this argument string to use Bazel's [Args object](https://bazel.build/rules/lib/Args) instead? This is mostly for performance reasons during the analysis phase and removes the need to call `.path` on all the file objects.

```suggestion
args = ctx.actions.args()
args.add_all(("otp", "alert-digest"))
args.add("--output", file)
args.add_all((ctx.file.otp_img, ctx.attr.lc_state))
```